kbase-extension/static/kbase/js/common/jobs.js

```diff
@@ -14,13 +213,23 @@ define([], () => {
      * @param {object} jobState
      */
     function isValidJobState(jobState) {
+        const requiredProperties = ['job_id', 'created'];
         if (jobState !== null && typeof jobState === 'object') {
-            return jobState.hasOwnProperty('created') && jobState.hasOwnProperty('job_id');
+            let isValid = true;
+            requiredProperties.forEach((prop) => {
+                if (!(prop in jobState)) {
+                    isValid = false;
+                }
+            });
+            return isValid;
         }
         return false;
     }
```

Comment:
could use `[].every()` / early return on the `if` here for conciseness

```js
if (jobState === null || typeof jobState !== 'object') return false;
const hasRequiredProps = requiredProperties.every(prop => jobState.hasOwnProperty(prop));
return hasRequiredProps;
```

Reply:
Totally forgot about `every` -- thanks!